templates/blog/macros.html

```diff
@@ -1,6 +1,8 @@
 {% macro article_list(articles, with_author=true) %}
+  <ul>
   {% for article in articles %}
   {% set url = url_for_article(article) %}
+  <li>
```

Comment:
nop, `articles` was printing an `<articles>`, @Ydrasil fix that in the template and this morning I fix the css (but all of that are on the ydrasi/neomad repo), It's going to be good :)